openfasoc/generators/ldo-gen/models/model_low_V.json

```diff
@@ -0,0 +1,104 @@
+{
+  "Iload,max": {
```

Comment:
why do we have 2 model files?

Reply:
Old model file. Removed with most recent commit.